apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/checkNoCapitalizedFields.kt

```diff
@@ -0,0 +1,85 @@
+package com.apollographql.apollo3.compiler
+
+import com.apollographql.apollo3.annotations.ApolloInternal
+import com.apollographql.apollo3.ast.GQLDefinition
+import com.apollographql.apollo3.ast.GQLField
+import com.apollographql.apollo3.ast.GQLFragmentDefinition
+import com.apollographql.apollo3.ast.GQLFragmentSpread
+import com.apollographql.apollo3.ast.GQLInlineFragment
+import com.apollographql.apollo3.ast.GQLOperationDefinition
+import com.apollographql.apollo3.ast.GQLSelection
+import com.apollographql.apollo3.ast.Issue
+import com.apollographql.apollo3.ast.internal.IssuesScope
+
+@ApolloInternal
+fun findCapitalizedFields(definitions: List<GQLDefinition>): List<Issue> {
```

Comment:
Super nit. I'd expect a method called "find" to return capitalized fields. Use "check" instead?

```suggestion
fun checkCapitalizedFields(definitions: List<GQLDefinition>): List<Issue> {
```

Reply:
Good point! I've renamed the 2 other `find` as well.